engine/runtime-language-epb/src/main/java/org/enso/interpreter/epb/EpbLanguage.java

```diff
@@ -2,42 +2,19 @@
 
 import com.oracle.truffle.api.CallTarget;
 import com.oracle.truffle.api.TruffleLanguage;
+import com.oracle.truffle.api.source.Source;
 import java.util.function.Consumer;
-import org.enso.interpreter.epb.node.ContextRewrapNode;
-import org.enso.interpreter.epb.node.ForeignEvalNode;
-import org.enso.polyglot.ForeignLanguage;
 
-/**
- * An internal language that serves as a bridge between Enso and other supported languages.
- *
- * <p>Truffle places a lot of emphasis on safety guarantees, which also means that single-threaded
- * languages cannot easily be called from multiple threads. We circumvent this by using two separate
- * {@link com.oracle.truffle.api.TruffleContext}s, one (often referred to as "outer") is allowed to
- * run Enso, Host Java, and possibly other thread-ready languages. Languages that cannot safely run
- * in a multithreaded environment are relegated to the other context (referred to as "inner"). The
- * inner context provides a GIL capability, ensuring that access to the single-threaded languages is
- * serialized.
- *
- * <p>This imposes certain limitations on data interchange between the contexts. In particular, it
- * is impossible to execute origin language's code when executing in the other context. Therefore
- * outer context values need to be specially wrapped before being passed (e.g. as arguments) to the
- * inner context, and inner context values need rewrapping for use in the outer context. See {@link
- * org.enso.interpreter.epb.runtime.PolyglotProxy} and {@link ContextRewrapNode} for details of how
- * and when this wrapping is done.
- *
- * <p>With the structure outlined above, EPB is the only language that is initialized in both inner
- * and outer contexts and thus it is very minimal. Its only role is to manage both contexts and
- * provide context-switching facilities.
- */
+/** An internal language that serves as a bridge between Enso and other supported languages. */
 @TruffleLanguage.Registration(
-    id = ForeignLanguage.ID,
+    id = "epb",
```

Comment:
`EpbLanguage` constant?

Reply:
Needless. There is only one use of such a constant. Others are in modules that don't compile against this module:

![image](https://github.com/enso-org/enso/assets/26887752/35ce1116-196d-45a2-bba5-b74412baaf61)

They wouldn't be able to use the constant anyway. The fact that this all holds together is ensured by tests.